index.html

```diff
@@ -278,18 +279,20 @@ <h6 class="description">Hello I’m a software developer! I can help you build a
     </section>
   </div>
   <footer class="form-cont" id="form-cont">
-  <form action="https://formspree.io/f/mrgdkyla" method="post">
+  <form action="https://formspree.io/f/mrgdkyla" method="post"  class="form">
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you place your error message close to the submit button.

<img width="947" alt="image" src="https://user-images.githubusercontent.com/52934128/197019293-2378972d-6eb6-416f-87b8-8543d0c706bd.png">

Reply:
- Hello @Tessyrich, your email validation is working fine, but the project requirements say that the error message should appear close to the submit button. I'd suggest you put the error message in the same `div` with the submit button and use the `flexbox` attribute to align them however you might please. Please try to fix this 👍 👍 

Example from your code:
 `<div class='btn-cont'><button type="submit" class="get-in see">Get in touch</button><small></small></div>`

CSS: `.btn-cont: {display: "flex";}`